torch/nn/modules/conv.py

```diff
@@ -996,3 +997,399 @@ def __init__(self, *args, **kwargs):
 # TODO: Conv2dLocal
 # TODO: Conv2dMap
 # TODO: ConvTranspose2dMap
+
+
+class _LazyConvXdMixin(LazyModuleMixin):
+
+    def reset_parameters(self) -> None:
+        if not self.has_uninitialized_params() and self.in_channels != 0:
+            super().reset_parameters()
+
+    def initialize_parameters(self, input) -> None:
+        if self.has_uninitialized_params():
+            with torch.no_grad():
```

Comment:
The reset_parameters function is already doing this no?

Reply:
Good catch! Fixed!